proprietary/tokens/src/brand/ams/color.tokens.json

```diff
@@ -1,21 +1,85 @@
 {
   "ams": {
     "color": {
-      "primary-black": { "value": "#000000" },
-      "primary-white": { "value": "#FFFFFF" },
-      "primary-blue": { "value": "#004699" },
-      "primary-red": { "value": "#EC0000" },
-      "dark-blue": { "value": "#102E62" },
-      "orange": { "value": "#FF9100" },
-      "yellow": { "value": "#FFE600" },
-      "green": { "value": "#BED200" },
-      "dark-green": { "value": "#00A03C" },
-      "blue": { "value": "#009DE6" },
-      "purple": { "value": "#A00078" },
-      "magenta": { "value": "#E50082" },
-      "neutral-grey1": { "value": "#E8E8E8" },
-      "neutral-grey2": { "value": "#BEBEBE" },
-      "neutral-grey3": { "value": "#767676" }
+      "border": {
+        "default": { "value": "#181818" }
+      },
+      "canvas": {
```

Comment:
If you do not expect variants such as hover you might not need the default:
```
      "canvas": { "value": "#ffffff" }
```
You did explain that you do this for consistency but I would argue that a single value doesn't need this nesting.

Reply:
I was not referring to the canvas color but the fact that you do not need a child called default if it's the only child. eg.

```
"error": {
	"default": { "value": "#ec0000" }
},
```

can be

```
"error": { "value": "#ec0000" },
```

Reply:
I recently saw a PR by @alimpens to rename default tokens in the styledictionary build. So this comment can probably be resolved.